vm/parser.ts

```diff
@@ -6,13 +6,54 @@
 module J2ME {
   declare var util;
   import assert = J2ME.Debug.assert;
+  import concat5 = StringUtilities.concat5;
+  import pushUnique = ArrayUtilities.pushUnique;
```

Comment:
Nit: this code doesn't actually use *pushUnique*, so it doesn't need to import it.